data/SonnenfeldEtal2013_Table3.csv

```diff
@@ -0,0 +1,26 @@
+#ID, zlens, vdisp, Reff, Mstar, zlens_err, vdisp_err, Reff_err, Mstar_err
+SL2SJ021247-055552, 0.750, 267, 8.92, 11.45, 0.001, 17, 0.01, 0.17
```

Comment:
Reff of 8.92 looks like an effective radius in kpc, not arcsec. Can you please investigate, and then correct the code below if necessary?